### PR TITLE
Translation improvements

### PR DESCRIPTION
**Pull request by spiccinini**
A "smart" translation of menu entries is added based on the _data/structure.yml and the titles and ref values of each page.
If you are in a "spanish" document then all the items that have been translated to spanish are shown.
This is not the best way to implement i18n, but I think it is an improvement.

Also added some spanish docs. The more important part is a new way to explain the configuration with more examples and focus on the Node / Community / Default configs. We should translate to english after some feedback?

### CODE REVIEW

**Review by G10h4ck — APPROVED** [review]

File: docs/es_config.txt [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+```
+
+=== Funcionamiento
+
+Al correr el comando `lime-config` lo primero que sucede es que se procesan los niveles *_Nodo_*,
+*_Comunidad_* y *_Default_* y se genera el archivo `/etc/config/lime-autogen`.
+Este archivo es el que contiene la configuración que realmente va a utilizarse por libremesh para configurar el sistema.
+
+
+The executable `lime-config` reads such file and writes all the resulting complex configuration in many other configuration files.
```
Comment (germanferrero) [inline docs/es_config.txt:96]: A bit of english text got merged here

File: docs/es_config.txt [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+.Opciones de sistema
+
+Las siguientes opciones de la seccion `system` son las más relevantes:
+----
+config lime system
+	option hostname 'LiMe-%M4%M5%M6'
+	option domain 'thisnode.info'
+----
+
+Te sugerimos que cambies el dominio +thisnode.info+ por el sitio de tu comunidad, por ejemplo +option domain 'altermundi.net'+.
```
Comment (germanferrero) [inline docs/es_config.txt:117]: This contradicts the suggestion in en_config. Which states:
We warmly suggest to change the domain from lan to a new subdomain of your community’s website, for example option domain mesh.altermundi.net. Beware that this domain (mesh.altermundi.net) should not already exist out of the network (i.e. is bad idea to use altermundi.net).

File: docs/es_config.txt [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+=============
+El código de país por defecto no viene especificado (entonces por lo tanto el defecto suele ser `US`),
+pero recomendamos encarecidamente que lo configuren. Si la red se encuentra en España entonces la configuración es
+`option country 'ES'`, lo que además permite el uso del canal 13 (permitido en España) que de otra forma está restringido.
+=============
+
+[NOTE]
+.distance
+=============
+Distance parameter indicates the maximum distance of the wireless connected devices.
```
Comment (germanferrero) [inline docs/es_config.txt:203]: Another bit of english

File: docs/es_config.txt [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+
+== Importante
+
+=== No editar otros archivos de configuración que no sean de libremesh
+
+La configuración de un nodo libremesh solo debe realizarse a travez de los archivos de configuración de libremesh
+y nunca editando directamente los otros archivos de configuración.
+Esto es así ya que los cambios realizados en archivos de configuración por fuera del sistema de
+configuración de libremesh se pueden perder cuando:
+* se realice un cambio en la configuración de libremesh
```
Comment (germanferrero) [inline docs/es_config.txt:317]: This doesn't show up as a list in the final web

**Review by germanferrero — COMMENTED** [review]
Thanks @spiccinini I have add some comments. I could make the fixes my self. Although I would need help to understand the advice related to the domain config

File: docs/es_config.txt (hunk 2 above)
Comment (spiccinini) [reply docs/es_config.txt:117]: Right, by mistake I translated an older version of the en_config docs

**Comment by spiccinini** [comment]
> Thanks @spiccinini I have add some comments. I could make the fixes my self. Although I would need help to understand the advice related to the domain config

Thanks! Please fix this if you can :)
I think that we can translate it back into the english version of en_config

**Pull request merged**